src/components/landing-hero/landing-hero.module.css

```diff
@@ -6,6 +6,10 @@
 	&.hasSubtitle {
 		align-items: flex-start;
 	}
+
+	@media (--dev-dot-tablet-up) {
+		margin-right: 25%;
+	}
```

Comment:
thought: I wonder how we could get this one to apply only when versions are present, like in `docs-view` via `.contentWithVersions`.

💭 Do you think we could add a non CSS modules class and reference it with `:global`

```css
&:global(.contentWithVersions) {
	@media (--dev-dot-tablet-up) {
		margin-right: 25%;
	}
}
```

Reply:
@BRKalow is this what you had in mind?
https://github.com/hashicorp/dev-portal/pull/945/commits/65b44bb7d3331cbcd2b5995f05720e42c6da0beb